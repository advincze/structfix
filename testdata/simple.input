package p

type foo struct {
	bar struct {
		baz int
	}
}

var _ = foo{bar: {baz:6}}
