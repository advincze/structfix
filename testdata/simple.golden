package foop

type foo struct {
	bar struct {
		baz int
	}
}

var _ = foo{bar: struct {
	baz int
}{baz: 6}}
